proposals/0007-live-edge.md

```diff
@@ -0,0 +1,196 @@
+- Feature Name: Live Edge
+- Start Date: 2023-02-08
+- PR: [video-dev/media-ui-extensions/#7](https://github.com/video-dev/media-ui-extensions/pull/7)
```

Comment:
It seems like `video-dev/media-ui-extensions#7` does auto-linkify. But I think either method works. We can pre-fill this link format to be updated.